app/components/SearchBox/index.js

```diff
@@ -33,7 +33,7 @@ class SearchBox extends Component {
   }
   getSuggestions(input, callback) {
     request
-    .get('https://tasks.hotosm.org/api/v1/project/search')
+    .get('https://tasks.hotosm.org/api/v2/projects/')
```

Comment:
https://tasks.hotosm.org/api/v2/projects returns a 404 for me. Are you sure this is the right API endpoint?

Reply:
Looks like it is `https://tasking-manager-tm4-production-api.hotosm.org/api/v2/…`.

Let me know if this one is only meant to be used internally, and what the real address would be.